src/CabanaPD_Particles.hpp

```diff
@@ -672,8 +675,8 @@ class Particles<MemorySpace, PMB, TemperatureDependent, Dimension>
 
     // These are split since weighted volume only needs to be communicated once
     // and dilatation only needs to be communicated for LPS.
-    using scalar_type = typename base_type::scalar_type;
-    using aosoa_temp_type = Cabana::AoSoA<scalar_type, memory_space, 1>;
+    using temp_types = Cabana::MemberTypes<double, double>;
```

Comment:
What is this change for?

Reply:
What is the difference between "temperature diffusion" and "current temperature"?